en/guide/migrating-5.md

```diff
@@ -126,6 +126,20 @@ In Express 4.7 and Express 5 onwards, the query parser option can accept `false`
 
 <h3>Improvements</h3>
 
+<h4 id="async.request.handlers">async request handlers</h4>
+
+Request handlers and middleware can now be be `async` functions, and they will behave in the same way as non-async handlers when an error occurs.
+
+In Express 4, it was necessary to explicitly handle rejected promises. Without handling the rejection, Node.js would exit and the client would receive an empty response. In Express 5, when middleware returns a rejected promise (for example, by using `await` inside an `async` request handler), Express behaves the same way as when an uncaught error is thrown: That is:
```

Comment:
> Without handling the rejection, Node.js would exit

Probably being too specific here, but Node.js didn't traditionally terminate the process on unhandled promises. Instead, it would print out an `UnhandledPromiseRejectionWarning`

Reply:
You're right.

I guess I can reword to say:

If the rejection was not handled, the client would receive an empty response (as opposed to an error), and as of Node 15 the `UnhandledPromiseRejectionWarning` would throw an error.

What do you think? Still too specific?

Reply:
Is there enough documentation around the default error handler that you can simply reference that it's the same behavior as if you throw an error or call `next(err)` inside of a route or middleware function? Instead of getting into all of these details, not the the details are bad!

Reply:
That's not a bad idea. Unsure about the details on the error handling docs, though.

Reply:
There is documentation around error handling here: http://expressjs.com/en/guide/error-handling.html

It covers the Express 5 behaviour, so I think I can link to it. I'll try to reword to make it a bit less verbose.

Reply:
I've simplified the wording of the whole block given that error handling is well covered in the error handling guide with 5f3ea21. Please let me know if what you think.

Reply:
Yeah I think the simplified version works well

Reply:
Will resolve. If you have any more comments on this section let me know.